.gitattributes

```diff
@@ -12,3 +12,4 @@
 # Tell GitHub this is generated code, and doesn't need to be shown in diffs by
 # default.
 cranelift/codegen/src/isa/x64/lower/isle/generated_code.rs linguist-generated
+cranelift/codegen/src/isa/aarch64/lower/isle/generated_code.rs linguist-generated
```

Comment:
Does the syntax support a wildcard (`.../isa/*/lower/isle/generated_code.rs`) here?